var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -178,7 +178,7 @@ class Mfem(Package):
     depends_on('unwind', when='+libunwind')
     depends_on('zlib', when='+gzstream')
     depends_on('gnutls', when='+gnutls')
-    depends_on('conduit@0.3.1:', when='+conduit')
+    depends_on('conduit@master:', when='+conduit')
```

Comment:
`master` is the oldest version in the package. With this, you're saying that the version doesn't matter. I think you want: `@0.3.1:,master`

Reply:
that makes sense, thanks for clarifying. I think I just want to allow master to be built. 
The problem is master is actually newer and @0.3.1: did not allow it to build.

Sounds like `@0.3.1:,master` will do it?

Reply:
Yep